backend/globaleaks/utils/utility.py

```diff
@@ -69,24 +92,29 @@ def read_json_file(p):
 
 
 def drop_privileges(user, uid, gid):
-    if os.getgid() != gid:
-        os.setgid(gid)
-        os.initgroups(user, gid)
-
-    if os.getuid() != uid:
-        os.setuid(uid)
+    if platform.system() != 'Windows':
+        if os.getgid() != gid:
+            os.setgid(gid)
+            os.initgroups(user, gid)
 
+        if os.getuid() != uid:
+            os.setuid(uid)
+    else:
+        log.err("Unable to securely drop permissions on Windows")
```

Comment:
No, it runs as non-administrator though Windows will require an admin to start it to allow firewall exceptions to be added, but it can't drop privileges below that point. This warning was added primarily to make sure I didn't forget to do something here at a later point.